include/glvnd/libglxabi.h

```diff
@@ -300,25 +300,30 @@ typedef struct __GLXapiImportsRec {
     /*!
      * (OPTIONAL) Callbacks by which the vendor library may re-write libglvnd's
      * entrypoints at make current time, provided no other contexts are current
-     * and the TLS model supports this functionality.  This is a performance
+     * and the TLS model supports this functionality. This is a performance
      * optimization that may not be available at runtime; the vendor library
-     * must not depend on this functionality for correctness.  This should
-     * point to a statically-allocated structure, or NULL if unimplemented.
+     * must not depend on this functionality for correctness.
+     *
+     * Like the __GLXapiImports struct itself, this struct is allocated and
+     * zeroed by libGLX. The vendor library should assign function pointers
+     * to the members of this struct, but should not change the
+     * \c patchCallbacks pointer itself.
```

Comment:
Make it a constant pointer (as opposed to the original constant data), so that users get a lovely warning ;-) Otherwise this comment might go unnoticed ?